hardhat.config.ts

```diff
@@ -159,24 +159,31 @@ const config: HardhatUserConfig = {
   deploymentManager: {
     relationConfigMap,
     networks: {
-      fuji: fujiRelationConfigMap,
-      kovan: kovanRelationConfigMap,
+      fuji: {
+        "USDC-01": fujiUSDCRelationConfigMap,
+      },
+      kovan: {
+        "USDC-01": kovanUSDCRelationConfigMap,
+      }
     },
   },
 
   scenario: {
     bases: [
       {
         name: 'development',
+        market: '01-USDC' // ?
```

Comment:
this is slightly awkward. I updated `ForkSpec` to require a `market` argument as well.

it seems like `market` should be required, but it means we need to add a market for the `development` base that will just be ignored.